presto-docs/src/main/sphinx/functions/tdigest.rst

```diff
@@ -0,0 +1,45 @@
+=========================
+T-Digest Functions
+=========================
+
+Data Structures
+---------------
+
+A T-digest is a data sketch which stores approximate percentile
+information.  The presto type for this data structure is called ``tdigest``.
+T-digests may be merged without losing precision, and for storage
+and retrieval they may be cast to/from ``VARBINARY``.
+
+Functions
+---------
+
+.. function:: merge(tdigest) -> tdigest
+    :noindex:
```

Comment:
Why `:noindex:` for all of these?

Reply:
This is weird. Apparently, I can remove one arbitrary `:noindex:` but it I remove two, it fails.